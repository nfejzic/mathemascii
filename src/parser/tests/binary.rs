use super::Snapshot;

super::test_snap!(frac, "frac 1.44 2 frac(x)(3.14)");
super::test_snap!(ubrace, "ubrace(1+2+3+4)_(\"4 terms\")");
super::test_snap!(obrace, "obrace(1+2+3+4)_(\"4 terms\")");
