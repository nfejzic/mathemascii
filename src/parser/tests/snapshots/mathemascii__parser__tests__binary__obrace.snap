---
source: src/parser/tests/binary.rs
expression: "Snapshot((input, math))"
---
obrace(1+2+3+4)_("4 terms")

Expression {
| Overbrace(
| | OpenParen
| | | Expression {
| | | | Number("1")
| | | }
| | | 
| | | Expression {
| | | | Operator(Plus)
| | | }
| | | 
| | | Expression {
| | | | Number("2")
| | | }
| | | 
| | | Expression {
| | | | Operator(Plus)
| | | }
| | | 
| | | Expression {
| | | | Number("3")
| | | }
| | | 
| | | Expression {
| | | | Operator(Plus)
| | | }
| | | 
| | | Expression {
| | | | Number("4")
| | | }
| | |
| | CloseParen
| )
| | sub:
| | | OpenParen
| | | | Expression {
| | | | | Text("4 terms")
| | | | }
| | | |
| | | CloseParen
}


